bzt/modules/python/generators.py

```diff
@@ -60,14 +60,16 @@ def create_method_name(label):
 
 
 class JMeterExprCompiler(object):
-    def __init__(self, parent_log):
+    def __init__(self, scenario, parent_log):
```

Comment:
It seems that JMeterExprCompiler doesn't really use scenario anywhere. Does it really need it?